sdk/core/azure-core/azure/core/pipeline/policies/_authentication.py

```diff
@@ -98,11 +97,14 @@ def on_request(self, request: PipelineRequest[HTTPRequestType]) -> None:
         self._enforce_https(request)
 
         if self._token is None or self._need_new_token:
-            if self._enable_cae:
-                self._token = self._credential.get_token(*self._scopes, enable_cae=self._enable_cae)
+            if hasattr(self._credential, "get_token_info"):
```

Comment:
Should we ways fall back to use get_token? e.g. if user specifies is_pop, we should only try get_token_info.

Reply:
Or the plan is we will add that later and identity with pop will depend on 1.32.0?

Reply:
From what I recall, if a token is PoP, then it'll need to be handled differently, likely in a different policy (e.g. `PopTokenCredentialPolicy`). I think the fallback behavior here is fine. Users wanting PoP tokens would need to be using the latest version of identity which would have all credentials implementing `get_token_info`.